package.yaml

```diff
@@ -23,6 +23,7 @@ dependencies:
 - base >= 4.7 && < 5
 - parsec
 - hspec
+- HUnit
```

Comment:
@JhaelArce Please update your branch to develop. You had files which are not required and we will overwrite them if I merge it.